stripe-identity-react-native.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 # When stripe_version is updated, also need to update stripe_version in https://github.com/stripe/stripe-react-native/blob/master/stripe-react-native.podspec
-stripe_version = '~> 23.8.0'
+stripe_version = '~> 23.10.0'
```

Comment:
Seems we should update this to `23.11.0`
https://github.com/stripe/stripe-ios/releases/tag/23.11.0

Reply:
if we do that we won't have a corresponding version for 0.29.0 of stripe-react-native: https://github.com/stripe/stripe-react-native/blob/master/stripe-react-native.podspec#L5 (we had to run a release already)

Reply:
ack, could you merge this and create a PR with `23.11.0`

Reply:
as discussed offline, will drop this PR and release with the latest native versions in https://github.com/stripe/stripe-identity-react-native/pull/131